.github/workflows/update-viablestrict.yml

```diff
@@ -0,0 +1,28 @@
+name: Update viable/strict
+
+on:
+  schedule:
+    # Run every 30 minutes at 10 and 40 minute marks
+    - cron: 10,40 * * * *
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: false
+
+jobs:
+  do_update_viablestrict:
+    if: ${{ github.repository_owner == 'pytorch' }}
+    runs-on: ubuntu-latest
+    environment: ${{ (github.event_name == 'schedule') && 'update-viable-strict' || '' }}
+    steps:
+      - name: Update viable/strict
+        uses: pytorch/test-infra/.github/actions/update-viablestrict@main
+        with:
+          repository: pytorch/vision
+          stable-branch: viable/strict
+          requires: '[\"Build Linux\", \"Build M1\", \"Build Macos\", \"Build Windows\", \"Tests\", \"CMake\", \"Lint\", \"Docs\"]'
```

Comment:
Can we just remove the macOS + Windows stuff here?

Is it really necessary?

Reply:
Yeah, let me tentatively make that change unless @NicolasHug wants to keep them.  `MacOS` (x86) is dead and should be removed anyway, but `Windows` is still a thing from what I see in https://github.com/pytorch/vision/actions